lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
@remear @NullVoxPopuli we got a [response from jsonapi](https://github.com/json-api/json-api/issues/1092) looks like the schema is incorrect in this case and we do need one of data/meta/links. Can't be null or {}.